src/emailservice/Dockerfile

```diff
@@ -12,28 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM python:3.10-slim
+FROM ruby:3.1.2
```

Comment:
Could a slim version of this image be used?

Reply:
@mic-max I had problems installing some gems (`puma` and `thin` both, I think) when I used the `-slim` image. I could either:
- Use a different webserver that's pure-ruby (`webrick`, maybe 😆 ), the tradeoff being they're not super performant. Not sure that it matters though.
- Manually add whatever packages are needed to get `puma` or `thin` to install.

Do you have any preference for which option is better here? I can definitely switch to a `-slim` image either way, though.

Reply:
Oh whoops, we merged. We can fix it in post. 😂

Reply:
Yeah my b haha. Please open a pr & I’ll review asap